ArduPlane/Parameters.cpp

```diff
@@ -852,7 +852,7 @@ const AP_Param::Info Plane::var_info[] = {
     // @Param: CRASH_DETECT
     // @DisplayName: Crash Detection
     // @Description: Automatically detect a crash during AUTO flight and perform the bitmask selected action(s). Disarm will turn off motor for safety and to help against burning out ESC and motor. Set to 0 to disable crash detection.
-    // @Bitmask: 0:Disarm
+    // @Bitmask: 0:Disable,1:Disarm
```

Comment:
The pre exsisting docs were really bad here. Bit 0 isn't disable, a value of 0 is disable. Bit 0 should actually be the disarm bit.

Reply:
@WickedShell sorry I got confused there. Please see the latest change and tell me if anything else needs to be changed!